kafka/kafka_url.go

```diff
@@ -31,6 +31,11 @@ func newKafkaSink(u *url.URL) (substrate.AsyncMessageSink, error) {
 
 	conf.Brokers = append(conf.Brokers, q["broker"]...)
 
+	v := q.Get("version")
```

Comment:
Nit: Since empty string is the default value anyway, you can drop the emptiness check
```suggestion
	conf.Version = q.Get("version")
```